src/platform/packages/shared/kbn-es/src/utils/docker_uiam.ts

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { MOCK_IDP_UIAM_SHARED_SECRET, MOCK_IDP_UIAM_SIGNING_SECRET } from '@kbn/mock-idp-utils';
+import type { ToolingLog } from '@kbn/tooling-log';
+import chalk from 'chalk';
+import execa from 'execa';
+import { setTimeout as setTimeoutAsync } from 'timers/promises';
+import { Agent } from 'undici';
+import { createHmac } from 'crypto';
+import { SERVERLESS_UIAM_ENTRYPOINT_PATH } from '../paths';
+
+const COSMOS_DB_EMULATOR_DOCKER_REGISTRY = 'mcr.microsoft.com';
+const COSMOS_DB_EMULATOR_DOCKER_REPO = `${COSMOS_DB_EMULATOR_DOCKER_REGISTRY}/cosmosdb/linux/azure-cosmos-emulator`;
+const COSMOS_DB_EMULATOR_DOCKER_LATEST_VERIFIED_TAG = 'vnext-preview';
```

Comment:
__note:__ we might need to pin to a specific SHA instead of `vnext-preview` to make sure new release doesn't suddenly breaks our tests.

Reply:
Done